tasks/configure.yml

```diff
@@ -17,8 +25,8 @@
   service: name={{ mongodb_daemon_name }} state=started enabled=yes
 
 - name: wait MongoDB port is listening
-  wait_for: host="{{ mongodb_conf_bind_ip }}"port="{{ mongodb_conf_port }}" delay=10 timeout=60 state=started
-  when: "'systemd' in systemd.stdout"
+  wait_for: host="{{ mongodb_listen_iface }}" port="{{ mongodb_conf_port }}" delay=5 timeout=60 state=started
+  #when: "'systemd' in systemd.stdout" #TODO Why only with systemd?
```

Comment:
Maybe then it will make sense to add a variable like `mongodb_always_wait_for_port` and change the condition to
```
  when: "'systemd' in systemd.stdout" or mongodb_always_wait_for_port
```

What do you think?